example/X_NUCLEO_IKS02A1_Python_Wave_Serial_Encoder/X_NUCLEO_IKS02A1_Python_Wave_Serial_Encoder.ino

```diff
@@ -0,0 +1,90 @@
+/* This example was developed for Nucleo F401RE or L476RG only. If you want to use other Nucleo or Architecture 
+ * you have to implement the dedicated file to manage the low level PDM part
+ */
+#if !defined(ARDUINO_NUCLEO_L476RG) && !defined(ARDUINO_NUCLEO_F401RE)
+#error "This example is only for STM32 Nucleo-F401RE or Nucleo-L476RG!"
+#endif
+
+#include "Arduino.h"
+#include "PDM.h"
+#include "pdm2pcm.h"
+#include <CMSIS_DSP.h>
+
+/* PDM */
+uint16_t sampleBuffer[(((AUDIO_IN_FREQ / 8) * MAX_AUDIO_IN_CHANNEL_NBR_PER_IF * N_MS_PER_INTERRUPT)/2)] = {0};
+uint32_t counter = 0;
+
+volatile bool recording = false;
```

Comment:
Hi @sarachiodo ,
you can remove volatile keyword from this declaration because this variable is not used in a ISR.